force-app/main/default/lwc/recordPickerHello/recordPickerHello.js

```diff
@@ -0,0 +1,73 @@
+import { wire, LightningElement } from 'lwc';
+import { gql, graphql } from 'lightning/uiGraphQLApi';
+
+export default class RecordPickerHello extends LightningElement {
+    selectedRecordId = '';
+    contact;
+
+    get variables() {
+        return {
+            selectedRecordId: this.selectedRecordId
+        };
+    }
+
+    handleChange(event) {
+        this.selectedRecordId = event.detail.recordId;
+    }
+
+    @wire(graphql, {
+        query: gql`
+            query searchContacts($selectedRecordId: ID) {
+                uiapi {
+                    query {
+                        Contact(
+                            where: { Id: { eq: $selectedRecordId } }
+                            first: 1
+                        ) {
+                            edges {
+                                node {
+                                    Id
+                                    Name {
+                                        value
+                                    }
+                                    Phone {
+                                        value
+                                        displayValue
+                                    }
+                                    Title {
+                                        value
+                                    }
+                                    Picture__c {
+                                        value
+                                        displayValue
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        `,
+        variables: '$variables'
+    })
+    wiredGraphQL({ data, errors }) {
+        if (errors) {
+            this.dispatchEvent(new CustomEvent('error', { error: errors }));
```

Comment:
Same feedback as for the other component: let's handle the error within the component.